docs/project/changelog.md

```diff
@@ -33,7 +41,11 @@ substitutions:
 
 - {{Fix}} Webpack building compatibility issues and a {any}`loadPyodide <globalThis.loadPyodide>`
   runtime issue due to webpack are solved. 
-  {pr}`1900`  
+  {pr}`1900`
+
+- {{API}} {any}`loadPyodide <globalThis.loadPyodide>` now accepts `homeDirectory`
```

Comment:
I wonder if should name this argument `home` (name of UNIX [$HOME](https://en.wikipedia.org/wiki/Home_directory) also in [pathlib.Path.home()](https://docs.python.org/3/library/pathlib.html#pathlib.Path.home)) or maybe better `homedir` as in [Node.js](https://nodejs.org/api/os.html#os_os_homedir). `homeDirectory` is a bit long.